conans/client/command.py

```diff
@@ -224,7 +224,9 @@ def new(self, *args):
                         circleci_gcc_versions=args.ci_circleci_gcc,
                         circleci_clang_versions=args.ci_circleci_clang,
                         circleci_osx_versions=args.ci_circleci_osx,
-                        template=args.template)
+                        template=args.template,
+                        template_dir=args.template_dir
```

Comment:
There is no longer a `args.template_dir`, isn't it?

Reply:
Fixed